about/strategy.md

```diff
@@ -1,30 +1,56 @@
 # Strategy and goals
 
-The 2i2c strategic plan is a combination of active and aspirational priorities that help us get closer to achieving our organizational mission. These are broken down into **strategic goals and objects for each**, with links to projects that feed into these objectives. There are four categories, associated with the [mission of 2i2c](https://2i2c.org/#what-we-do):
+Our Strategic Plan is a combination of active and aspirational priorities that help us get closer to achieving our organizational mission. These are broken down into **strategic goals and objects for each**. The [major projects of 2i2c](../reference/projects.md) should all feed into this strategy.
 
-- {badge}`research,badge-primary`: Interactive Computing in Research
-- {badge}`education, badge-secondary`: Supporting Interactive Computing in Education
-- {badge}`open-source, badge-info`: Supporting Open Source Communities for Interactive Computing
-- {badge}`organization, badge-success`: Organizational and operational goals
+Currently, 2i2c's Strategic Plan consists of a 1-year "bootstrap strategy".
+Our primary goal is to build a foundation for 2i2c and clarify out strategy and objectives for the coming years.
+This plan was created by the Founding Team of 2i2c [and is located here](https://docs.google.com/document/d/13uxWKWMMAdvM-knC5fMOj8tP_HQVvijIIE9jZgTItX4/edit?usp=sharing).
```

Comment:
@choldgraf I can edit this doc as incognito, maybe it has the wrong permissions?

Reply:
Good point, will switch to view only unless explicitly added